### PR TITLE
[REF] beesdoo_stock: split module

### PR DESCRIPTION
**Pull request by robinkeunen**
https://github.com/OCA/stock-logistics-workflow/pull/1119

### CODE REVIEW

File: beesdoo_stock/models/stock.py [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-                "view_mode": "list",
-                "res_model": "product.template",
-                "view_id": False,
-                "target": "current",
-                "type": "ir.actions.act_window",
-                "context": ctx,
-                "nodestroy": True,
-                "res_id": ctx["articles"],
-                "domain": [("id", "in", ctx["articles"])],
-            }
```
Comment (victor-champonnois) [inline beesdoo_stock/models/stock.py:28]: I don't understand, this is not the same code as in https://github.com/OCA/stock-logistics-workflow/pull/1119/files#diff-df167430c1f8baf5c14a15db17f7fc03019412c41df18fee8042b289c605ffc2. Is it just a split or did you change some stuff ?

Comment (robinkeunen) [reply beesdoo_stock/models/stock.py:28]: I recoded it the proper way indeed.

**Review by victor-champonnois — APPROVED** [review]
LGTM

**Pull request merged**